ui/cypress/CYPRESS.md

```diff
@@ -0,0 +1,66 @@
+# Cypress Testing
+
+Cypress is being used for both end-to-end tests and component tests.
+
+## Prerequisites
+
+1. You have an AAP instance being deployed on Azure.
+2. The Deployment Engine UI is accessible.
+
+## Run
+
+1. Initialize your local environment.
+   You may need to create a virtualenv and install the needed packages from ui/requirements.txt if you don't already have them.
+
+    ```shell
+    cd <repo>/ui
+    virtualenv venv
+    source venv/bin/activate
+    pip install -r requirements.txt
```

Comment:
What is the reason for python environment? We have no python code in this repo except for one python script that's used in development only.

Reply:
Normally people probably don't have flask in their system python env (I don't), so a virtualenv is the normal way to add the stuff from requirements.txt (including flask) and run it.  The note was already there in your PR, but I added the steps.

Reply:
Fine to remove it and let people figure it out though if you like!

Reply:
Having a python virtual environment to install dependencies will not mess up our OS level installations. I prefer to keep it in our documentation. I will add a note to say it is optional.